Dangerfile

```diff
@@ -1,6 +1,14 @@
 # Warn if a pull request is too big
 MAX_PR_SIZE = 250
-if git.lines_of_code > MAX_PR_SIZE
+EXCLUSIONS = ['Gemfile.lock', '.json', 'spec/fixtures/', '.txt']
```

Comment:
Should we add `.yml` files?

Reply:
My first thought is no because things like `Settings.yml` & `docker-compose.yml` _are_ things I'd want to include. However, you made me realize that our VCR cassettes are YAML 😱 . I'll add an exception for our cassettes as a middle ground.